pkg/collector/corechecks/containers/containerd.go

```diff
@@ -195,7 +196,7 @@ func computeMetrics(sender aggregator.Sender, nk context.Context, cu cutil.Conta
 		}
 		tags = append(tags, userTags...)
 		// Tagger tags
-		taggerTags, err := tagger.Tag(ctn.ID(), false)
+		taggerTags, err := tagger.Tag(ctn.ID(), collectors.HighCardinality)
```

Comment:
shouldn't this be low to match false?

Reply:
Setting it to false was a mistake, Charly asked me to update it to HighCard while I was touching the file.